forwards_paper.tex

```diff
@@ -236,31 +236,27 @@ \subsection*{Simulation benchmarks}
 With \simupop{}, we were only able to run simulations with $N=10^3$ and region sizes up to $10^4$.  Pedigree tracking
 dramatically improves run times for both \fwdpy{} and \simupop{} (right column of Figure~\ref{fig:runtimes_selection}).  For parameters that ran to completion when tracking neutral mutations and when tracking
-the pedigree, the relative speedup due to pedigree tracking is up to $\approx 50$ fold (Figure~\ref{fig:relative_speedup_selection}).  For \fwdpy{}, pedigree tracking reduces runtimes more for larger population sizes
-(Figure~\ref{fig:relative_speedup_selection}).  We could not make such an observation with \simupop{}, which tended to
-run out of memory on our system.  Note the log scale of the x axis in Figure~\ref{fig:runtimes_selection}, which
-partially obscures an important observation that the run times appear approximately linear in region size when pedigree
-tracking.  The reason for this behavior is that the simulations are generating a constant number of new edges on average
-each generation, and the number of edges is a function of $\rho$ in expectation.
-
-We also performed a more limited set of simulations without natural selection.  The total run times are shown in
-Figure~\ref{sfig:rawspeed_nosel} and show the same qualitative behavior as simulations with selection
-(Figure~\ref{fig:runtimes_selection}).  The relative improvement due to pedigree tracking is again substantial in these
-simulations (Figure~\ref{sfig:speedup_nosel}).  In fact, we see more of a benefit to pedigree tracking here with \fwdpy{}
-than we did with selection (Figure~\ref{fig:relative_speedup_selection}).  The reason is that the fitness function exits
-close to instantly in simulations without selection that are based on \fwdpp{}, meaning that \fwdpy{} is doing little
-more than generating random numbers and book-keeping in Figures~\ref{sfig:rawspeed_nosel}
-and~\ref{sfig:speedup_nosel}.
+the pedigree, the relative speedup due to pedigree tracking is up to $\approx 50$ fold
+(Figure~\ref{fig:relative_speedup_selection}).  For \fwdpy{}, pedigree tracking reduces runtimes more for larger
+population sizes and we observe increasing relative speedups as $4Nr$ increases for a given $N$
+(Figure~\ref{fig:relative_speedup_selection}).  We saw a similar trend with $4Nr$ for \simupop{}, but the effect of $N$
+could not be explored because the simulations tended to run out of available memory.
+  Note the log scale of the x axis in Figure~\ref{fig:runtimes_selection}, which
+partially obscures an important observation that the run times appear approximately linear in region size when tracking
+pedigree.  The reason for this behavior is that, when tracking pedigrees, the simulations are generating a constant number of new edges on
+average
+each generation as the number of new recombination breakpoints is Poisson distributed with expectation $\rho$.
```

Comment:
Rather than have paragraphs referring to supplement, maybe summarize briefly in two sentences L273-280 and L290-300 here. This makes it easier on the reader, I think. If we need these details in the main text maybe the figures should be here too. Something like:

> Our method uses more RAM use than simple forward simulations (see appendix B.1), although in practice shorter simplification intervals may reduce RAM use (at a slight cost to the speed up achieved, see \subsubsection*{Sequential simplification and prior history}). The benefits to our method are not limited to situations with selection, in a more limited set of neutral simulations we found the same qualitative behavior (see appendix B.2).